shpc/main/modules/__init__.py

```diff
@@ -343,7 +357,24 @@ def check(self, module_name):
         config = self._load_container(module_name.rsplit(":", 1)[0])
         return self.container.check(module_name, config)
 
-    def install(self, name, tag=None, symlink=False, **kwargs):
+    def write_version_file(self, version_dir):
```

Comment:
Is this section old from before? I guess we need to merge the version changes before this one!

Reply:
Yes, because I had done a merge of the `tcl_default_version` :/
I can probably remove this merge commit to make this PR clearer ?

Reply:
Yes - let's update this PR with the changes in current main and then I can give another review!